module/module.go

```diff
@@ -107,17 +107,8 @@ func (m *Module) GetDownloadName() string {
 }
 
 // GetBuildPath returns the path to the please BUILD file where this module is defined.
-func (m *Module) GetBuildPath() string {
-	splitPath := strings.Split(m.Path, "/")
-	pathMinusEnd := strings.Join(splitPath[:len(splitPath)-1], "/")
-	if splitPath[0] == "github.com" {
-		pathMinusEnd = strings.Join(splitPath[:2], "/")
-	}
-	currentDir, err := os.Getwd()
-	if err != nil {
-		panic(err)
-	}
-	return fmt.Sprintf("%s/third_party/go/%s/BUILD", currentDir, pathMinusEnd)
+func (m *Module) GetBuildPath(thirdParty string) string {
+	return filepath.Join(thirdParty, filepath.Dir(m.Path), "BUILD")
```

Comment:
This is now relative but file operations are relative to the working directory anyway. This means we shouldn't `os.Chdir()` anywhere in our code but that seems reasonable.